src/tests/ftest/util/telemetry_utils.py

```diff
@@ -449,6 +449,29 @@ def get_container_metrics(self):
                         data[host][name] = metric["value"]
         return data
 
+    def get_io_metrics(self, test_metrics=None):
+        """Get the container telemetry test metrics.
```

Comment:
I'm not familiar with telemetry, but does this method return data about some container's telemetry? If so, why it doesn't take container as parameter or use it in the method?

Reply:
Please see update. 
The method return a dict for the specified io telemetry metrics.
Thanks.